bespin/settings.py

```diff
@@ -138,5 +138,21 @@
         'rest_framework.authentication.BasicAuthentication',
         'rest_framework.authentication.TokenAuthentication',
         'rest_framework.authentication.SessionAuthentication',
-    )
-}
\ No newline at end of file
+    ),
+    'PAGE_SIZE': 20,
+    'EXCEPTION_HANDLER': 'rest_framework_json_api.exceptions.exception_handler',
```

Comment:
Good catch. I'll see what we can do here.

API clients (like curl) still get a 401 response.

With ` 'EXCEPTION_HANDLER': 'rest_framework_json_api.exceptions.exception_handler',`:

```
$ curl http://127.0.0.1:8000/api/jobs/
[{"status":"401","source":{"pointer":"/data"},"detail":"Authentication credentials were not provided."}]
```

Without:

```
$ curl http://127.0.0.1:8000/api/jobs/
{"detail":"Authentication credentials were not provided."}
```